rcl/src/rcl/timer.c

```diff
@@ -309,6 +309,9 @@ rcl_timer_get_time_until_next_call(const rcl_timer_t * timer, int64_t * time_unt
 {
   RCL_CHECK_ARGUMENT_FOR_NULL(timer, RCL_RET_INVALID_ARGUMENT);
   RCL_CHECK_ARGUMENT_FOR_NULL(time_until_next_call, RCL_RET_INVALID_ARGUMENT);
+  if (rcutils_atomic_load_bool(&timer->impl->canceled)) {
+    return RCL_RET_TIMER_CANCELED;
```

Comment:
We also need to update the doc in header file to describe it possibly returns `RCL_RET_TIMER_CANCELED` when the time has been canceled?

Reply:
Done in d73a3ea